buildout.cfg

```diff
@@ -1,2 +1,2 @@
 [buildout]
-extends = test-4.3.x.cfg
+extends = test-5.0.x.cfg
```

Comment:
ops.. tenho que tirar isso..

Reply:
Why 5.0 not 5.1?

Reply:
ah, sorry, now I see, its for 1.x!

Reply:
@hvelarde my mistake.. I thought it was one of my PR in another project.